dcpy/lifecycle/scripts/package_and_distribute.py

```diff
@@ -93,11 +125,18 @@ def from_bytes_to_tagged_socrata_cli(
         help="Only push metadata (including attachments).",
     ),
 ):
-    results = from_bytes_to_tagged_socrata(
+    datasets_set: set[str] = set()
+    if datasets:  # TODO: can typer do this natively?
```

Comment:
So this can be done natively - see https://github.com/NYCPlanning/data-engineering/blob/main/dcpy/lifecycle/scripts/validate_ingest.py#L235. Then can be called with `python3 -m ... -k boro -k block -k lot` which would be read by typer as `["boro", "block", "lot"]`

Reply:
I don't know that you need to do that here though

Reply:
right... so, I think if that's what typer supports, it's probably best to just go with that. Will change